tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "noImplicitAny": true,
+    "suppressImplicitAnyIndexErrors":true,
```

Comment:
@amansharma0091 removing this while merging
I have hack interface for this use case IAttribute
use like this `return (<IAttribute>this)[key + 'Text']`